core/src/database/franchise/franchise_profile/franchise_profile.model.ts

```diff
@@ -18,13 +19,24 @@ export class FranchiseProfile extends BaseModel {
     @Field(() => String)
     code: string;
 
-    @Column({nullable: true})
-    @Field({nullable: true})
-    scrimReportWebhookUrl?: string;
+    // Scrim Report Cards
+    @ManyToOne(() => Webhook, {nullable: true})
+    @Field(() => Webhook, {nullable: true})
+    scrimReportCardWebhook?: Webhook;
+
+    // League Play Report Cards
+    @ManyToOne(() => Webhook, {nullable: true})
+    @Field(() => Webhook, {nullable: true})
+    matchReportCardWebhook?: Webhook;
+
+    // Submissions Ready for Ratification or Failed
+    @ManyToOne(() => Webhook, {nullable: true})
+    @Field(() => Webhook, {nullable: true})
+    submissionWebhook?: Webhook;
 
     @Column({nullable: true})
-    @Field({nullable: true})
-    matchReportWebhookUrl?: string;
+    @Field(() => String, {nullable: true})
+    submissionDiscordRoleId?: string;
```

Comment:
Lets call this `discordRoleId` because presumably it could be used for something other than submissions